prep-html-syntax/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+  <html>
+    <body>
```

Comment:
Hi Yuliia, be sure to include a `<head>` and `<title>` in your document so that it is complete.